app/blog/tanstack-router-typescript-performance.md

```diff
@@ -0,0 +1,214 @@
+---
+title: A milestone for TypeScript Performance in TanStack Router
```

Comment:
Please add a published date, like with other articles.

Reply:
Will when we decide on the date to publish ;)